testdata/ruby/transpile/balancedBrackets/implementation/BalancedBrackets.java

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) rosettacode.org
```

Comment:
These are just becoming duplicated files. We should make them symlinks at some point maybe? Added https://github.com/symflower/eval-dev-quality/issues/325.

Reply:
Because then I would not have to review these by hand ^^